### PR TITLE
Release 7.8.0

### PR DESCRIPTION
**Pull request by tolusha**
Changes done during the release should be merged to master because they contain new OLM files and new default GO values that will be used as the new basis for future master changes.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -10,32 +10,36 @@
...
-olm/testUpdates.sh openshift stable
+olm/testUpdate.sh openshift stable
 ```
 
-Open Eclipse Che dashboard in the anonymous to validate that the correct version is installed and workspace can be created:
+Open Eclipse Che dashboard in an anonymous tab:
 
 ```bash
-echo http://$(kubectl get ingress -n che | grep ^che | awk -F ' ' '{ print $2 }')
+echo http://$(oc get route -n eclipse-che-preview-test | grep ^che | awk -F ' ' '{ print $2 }')
```
Comment (davidfestal) [inline RELEASE.md]: The even simpler solution is to retrieve the URL from the status of the `CheCluster` CR when it is in the `Available` state.

File: make-release.sh [hunk 2]
```diff
@@ -37,17 +42,17 @@ check() {
   command -v operator-sdk >/dev/null 2>&1 || { echo -e $RED"operator-sdk is not installed. Aborting."$NC; exit 1; }
 
   local operatorVersion=$(operator-sdk version)
-  [[ ! $operatorVersion =~ .*v0.10.0.* ]] || { echo -e $RED"operator-sdk v0.10.0 is required"$NC; exit 1; }
+  [[ $operatorVersion =~ .*v0.10.0.* ]] || { echo -e $RED"operator-sdk v0.10.0 is required"$NC; exit 1; }
```
Comment (davidfestal) [inline make-release.sh]: why this change ?

File: pkg/deploy/defaults.go [hunk 3]
```diff
@@ -69,10 +69,10 @@ const (
 	// For CRW these should be synced by hand with images stored in RH registries
 	// instead of being synced by script with the content of the upstream `che.properties` file
 	// NB:
-	// The upstream equivent are stored in the generated `extra_images.go` source file.
-	defaultCheWorkspacePluginBrokerInitImage    = "quay.io/crw/pluginbrokerinit-rhel8:2.0-6"
-	defaultCheWorkspacePluginBrokerUnifiedImage = "quay.io/crw/pluginbroker-rhel8:2.0-5"
-	defaultCheServerSecureExposerJwtProxyImage  = "quay.io/crw/jwtproxy-rhel8:2.0-4"
+	// The upstream equivalent are stored in the generated `extra_images.go` source file.
+	defaultCheWorkspacePluginBrokerMetadataImage    = "quay.io/crw/pluginbroker-rhel8:2.1"
```
Comment (davidfestal) [inline pkg/deploy/defaults.go]:
```suggestion
	defaultCheWorkspacePluginBrokerMetadataImage    = "quay.io/crw/pluginbroker-metadata-rhel8:2.1"
```

File: pkg/deploy/defaults.go [hunk 4]
```diff
@@ -69,10 +69,10 @@ const (
...
+	defaultCheWorkspacePluginBrokerArtifactsImage = "quay.io/crw/pluginbrokerinit-rhel8:2.1"
```
Comment (davidfestal) [inline pkg/deploy/defaults.go]:
```suggestion
	defaultCheWorkspacePluginBrokerArtifactsImage = "quay.io/crw/pluginbroker-artifacts-rhel8:2.1"
```

File: make-release.sh (hunk 2 above)
Comment (tolusha) [reply make-release.sh]: '||' uses to execute the code when the previous command failed so
`[[ $operatorVersion =~ .*v0.10.0.* ]] || {...` will be run when version isn't `v0.10.0`

File: RELEASE.md (hunk 1 above)
Comment (tolusha) [reply RELEASE.md]: Agree, I will update the script and use it in the next release.
